index.html

```diff
@@ -128,5 +130,23 @@ <h4>About me</h4>
         </ul>
     </div>
 </div>
+<div class="form">
+    <form action="https://formspree.io/f/mayknebl" method="post">
+        <p class="form_p">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <input type="text" name="fullname" placeholder="Full name" maxlength="30" required />
+        <input type="email" name="emailaddress" placeholder="Email address"  required />
+
+        <textarea  name="txt_area" placeholder="Enter text here" maxlength="500" required></textarea>
+        <button class="form_btn" type="submit">Get in Touch</button>
+    </form>
+    
+</div>
+<ul class="links">
+    <li class="link"><a href="#"><img src="images/links/github_b.svg" alt="github"></a></li>
+    <li class="link"><a href="#"><img src="images/links/linkedin_b.svg" alt="linkedin"></a></li>
+    <li class="link"><a href="#"><img src="images/links/angelist_b.svg" alt="angelist"></a></li>
+    <li class="link"><a href="#"><img src="images/links/twitter_b.svg" alt="twitter"></a></li>
+    <li class="link"><a href="#"><img src="images/links/medium_b.svg" alt="medium"></a></li>
+</ul>
```

Comment:
- [ ] Kindly insert the elements on the bottom of the web page inside the `<footer>` tag, for best practices for HTML & CSS.
- [ ] Please, 🙏🏽 add the line at the end of the web page to be similar to the Figma template as much as possible.  👇🏽 
- [ ] **[OPTIONAL]** Kindly replace the name in the logo "John Doe" with your name. 👍🏽 

- Figma template

![line](https://user-images.githubusercontent.com/58642949/186312426-861b0484-b101-4107-b4ed-3cc746224d1f.png)

- Your design

![no-line](https://user-images.githubusercontent.com/58642949/186312466-e4903fac-0d6e-4d8a-b5b7-86e640e9e432.png)